src/main/java/dal/fatih/todoly/Task.java

```diff
@@ -0,0 +1,73 @@
+package dal.fatih.todoly;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Task {
+
+    private String id;
+    private String title;
+    private String description;
+    private String date;
+    private static final AtomicInteger idGenerator = new AtomicInteger(1);
+
+
+    public Task(String title, String description, String date) throws ParseException {

```

Comment:
POJO classes shouldn't have too much logic in it. Convert the date to be Date and pass it down to this and keep this class as simple as possible